core/bin/dust_api.rs

```diff
@@ -121,8 +141,8 @@ impl APIState {
                 });
             });
             loop_count += 1;
-            tokio::time::sleep(std::time::Duration::from_millis(100)).await;
-            if loop_count % (10 * 10) == 0 {
+            tokio::time::sleep(std::time::Duration::from_millis(4)).await;
```

Comment:
This was potentially a bad source of latency move to 4 but we could also move it to even 1.